schema/crdb/12.0.1/up01.sql

```diff
@@ -0,0 +1,4 @@
+CREATE UNIQUE INDEX IF NOT EXISTS lookup_deleted_disk ON omicron.public.disk (
```

Comment:
With my apologies: I just merged a `13.0.0` schema migration, so I think this will have to become `14.0.0` (or maybe even `15.0.0` if #4545 lands first)

Reply:
This is a very annoying problem. @davepacheco’s idea of named migrations and a canonical order list is sounding pretty good to me!

Reply:
I'll wait for #4545, that's a higher priority.